query/expression.go

```diff
@@ -109,6 +110,21 @@ func (value scalarValue) toScalar() (float64, error) {
 	return float64(value), nil
 }
 
+// toDuration will take a value, convert it to a string, and then parse it.
+// It produces a nanosecond count as a signed int64.
+// the valid ns, us (µs), ms, s, m, h
+func toDuration(value value) (int64, error) {
```

Comment:
The formats are just those supported by `time.ParseDuration()` which represents time with nanosecond resolution... Currently I wasn't actually scaling at all, which would probably behave really strangely...

The comments for `api.Timerange` suggest that it's currently supposed to represent seconds, but will later represent milliseconds?

Has this been properly established yet?